app-connector-client/server/resources/api.yaml

```diff
@@ -1,20 +1,10 @@
 openapi: '3.0.0'

```

Comment:
Please provide example values for schema: EventDetails

Reply:
And also for the response of POST /events endpoint

Reply:
And the same for whole /api endpoint, please provide example values for request/response attributes